### PR TITLE
[PERFSCALE-2465] Updates the jobs in 4.14 EC Build

### PR DESCRIPTION
**Pull request by krishvoor**
This PR updates the data-path node count & adds a node-density-heavy job
for the EC Builds.

### CODE REVIEW

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-ec.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ tests:
     - chain: openshift-qe-data-path-tests
     workflow: rosa-aws-sts
   timeout: 5h0m0s
-- as: perfscale-rosa-multiaz-ec-ci-tests-2nodes-periodic
+- as: perfscale-rosa-multiaz-ec-ci-tests-24nodes-periodic
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-ec.yaml:57]: Can the test name here include `data path` or `control plane` or something of that sort for more clarity?

**Review by svetsa-rh — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-ec.yaml (hunk 1 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-ec.yaml:57]: @svetsa-rh Ack, once this PR gets merged - will change the PR details as well:
https://github.com/openshift/release/pull/42788

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 2]
```diff
@@ -2934,18 +2934,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-ec-x86-perfscale-rosa-multiaz-ec-control-plane-2nodes-periodic
-  reporter_config:
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:2938]: let's add this back in

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 3]
```diff
@@ -3021,18 +3010,83 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-ec-x86-perfscale-rosa-multiaz-ec-data-path-3nodes-periodic
-  reporter_config:
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3025]: and this reporter part as well

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml (hunk 2 above)
Comment (krishvoor) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:2938]: Ack, doing this now!

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml (hunk 3 above)
Comment (krishvoor) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3025]: Ack, adding this back

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-ec-x86-perfscale-rosa-multiaz-ec-control-plane-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-ec-x86-perfscale-rosa-multiaz-ec-node-density-heavy-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-ec-x86-perfscale-rosa-multiaz-ec-data-path-9nodes-periodic

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-ec-x86-perfscale-rosa-multiaz-ec-node-density-heavy-24nodes-periodic

**Pull request merged**